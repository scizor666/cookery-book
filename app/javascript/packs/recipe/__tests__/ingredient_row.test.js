import React from 'react';
import IngredientRow from '../ingredient_row';
import {ingredients} from './test_data';
import renderer from 'react-test-renderer';

test('display row ingredient', () => {
    const wrapper = renderer.create(<IngredientRow ingredient={ingredients[0]}/>);
    expect(wrapper.toJSON()).toMatchSnapshot();
});