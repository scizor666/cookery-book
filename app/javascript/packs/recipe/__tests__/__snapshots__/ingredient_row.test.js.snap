// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display row ingredient 1`] = `
<tr>
  <td
    className="ingredient-word-breaks"
  >
    chicken
  </td>
  <td>
    300
  </td>
  <td>
    107.7
  </td>
</tr>
`;
